src/ui/components/ShowMoreCard/index.js

```diff
@@ -55,6 +60,10 @@ export class ShowMoreCardBase extends React.Component {
         <a className="ShowMoreCard-revealMoreLink" href="#show-more"
           onClick={this.expandText}>
           {i18n.gettext('Show more info…')}
```

Comment:
Can this text be "Read more"? I find it confusing that there is a block below called "More information" and this says "Show more info...".